### PR TITLE
Add document

### PR DESCRIPTION
**Pull request by Marie-Garde**

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
Seems ok to me

File: src/components/UpdateRessources.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .post(`http://localhost:3001/ressources`, values)
+      .then((res) => {
+        setValues(res.data);
+      })
+      .catch((e) => {
+        console.warn(e);
```
Comment (LoicBrassart) [inline src/components/UpdateRessources.jsx]: Nope

**Review by LoicBrassart — CHANGES_REQUESTED** [review]

**Review by Thoms-C — APPROVED** [review]
The console.warn has been PURGED !

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**